Makefile

```diff
@@ -34,6 +34,13 @@
 SHELL := bash
 ENABLE_FEATURES ?=
 
+# Rust & C/C++ compiler flags
+#
+# Enable frame pointers for stable CPU Profiling.
+export RUSTFLAGS := $(RUSTFLAGS) -Cforce-frame-pointers=yes
```

Comment:
Maybe they should be modified in `etc/cargo.config.dist` like what I have done in https://github.com/tikv/tikv/pull/12326?

Reply:
Yes, though, we still need to set `build-std` in `cargo.config.dist`.

Reply:
Since we need to control whether frame-pointer and build-std are enabled through env vars, writing them in the configuration file is no longer suitable.